flink-connector-flume/pom.xml

```diff
@@ -36,7 +36,7 @@ under the License.
 	<!-- Allow users to pass custom connector versions -->
 	<properties>
 		<flume-ng.version>1.9.0</flume-ng.version>
-		<testcontainers.version>1.15.3</testcontainers.version>
+		<testcontainers.version>1.16.2</testcontainers.version>
```

Comment:
is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)